### PR TITLE
Add reshuffle-local-server helper script

### PR DESCRIPTION
**Pull request by vogre**
For running without create-react-app
Used in a package.json as { "start": "reshuffle-local-server" }

### CODE REVIEW

File: local-proxy/src/localServer.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+import { setupProxy } from './index';
+import program from 'commander';
+const NAME = 'reshuffle-local-server';
+program.option('-p, --port', 'listening port (3000 by default)');
+program.description(`Run the ${NAME} outside of create-react-app server`);
+
+function main() {
+  program.parse(process.argv);
+  // create the express app to route local requests
+  const app = express();
```
Comment (bergundy) [inline local-proxy/src/localServer.ts]: There should be a way for the user to configure his own routes and middleware for this to be useful.

File: local-proxy/src/localServer.ts [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+  // reshuffle currently assumes running from src and uses
+  // a relative path resolution to reach backend
+  setupProxy(process.cwd() + '/src')(app);
```
Comment (bergundy) [inline local-proxy/src/localServer.ts]: Does this work on windows? (just verifying)

File: local-proxy/src/localServer.ts (hunk 1 above)
Comment (bergundy) [reply local-proxy/src/localServer.ts]: Scratch this comment

**Comment by bergundy** [comment]
Can we add tests for this?

File: app-testsuite/src/run.ts [hunk 3]
```diff
@@ -1,12 +1,15 @@
...
+import { mkdtemp, readFile, writeFile } from 'mz/fs';
 import { tmpdir } from 'os';
 import { spawn as spawnChild, ChildProcess } from 'child_process';
 import { copy, remove, pathExists } from 'fs-extra';
 import { Tail } from 'tail';
 import sleep from 'sleep-promise';
 import { spawn, waitOnChild } from '@reshuffle/utils-subprocess';
+import got from 'got';
+import _assert from 'assert';
+const assert = _assert.strict;
```
Comment (bergundy) [inline app-testsuite/src/run.ts]: Or just `import { strict as assert } from 'assert';

File: local-proxy/src/localServer.ts (hunk 2 above)
Comment (bergundy) [reply local-proxy/src/localServer.ts]: @vogre PING

Comment (vogre) [reply local-proxy/src/localServer.ts]: Yep! It does produce a strange path at disk root (`C://\src`, if you decide that your project root is on `C:\\`, but even that works).

**Pull request merged**